### ISSUE TITLE
plotting clean-up

### ISSUE THREAD

**Issue by cboisvenue** [cboisvenue/spadesCBM#46]

**Comment by camillegiuliano** [comment]
Added to my list!
One thing: the link to Vini's stuff doesn't work on my end, it sends me to 404 Page not found error

**Comment by cboisvenue** [comment]
Yes. That is a private repo. I forked it. You can access it here: https://github.com/cboisvenue/rcbm4.git

**Comment by camillegiuliano** [comment]
I think you made yours private too, I'm getting the 404 error there too!
